pkg/cmdlets/auth/login.go

```diff
@@ -241,25 +240,9 @@ func AuthenticateWithBrowser(out output.Outputer, auth *authentication.Auth, pro
 		out.Notice(locale.Tr("err_browser_open", *response.VerificationURIComplete))
 	}
 
-	if !response.Nopoll {
```

Comment:
`NoPoll` was removed from the response type. No new fields were added. Not sure if this is how we want to handle this.